assets/_locales/es/messages.json

```diff
@@ -731,6 +731,10 @@
     "message": "Últimos enlaces de página usados",
     "description": "Last used page links"
   },
+  "settings_anilist_update_ui": {
+    "message": "Actualizar UI",
```

Comment:
The option adds a section to directly update an entry. I am not sure but something in the direction "UI de/para editar" is probably better here. Or what do you think?
![image](https://user-images.githubusercontent.com/12599334/171608493-84d037b2-42f3-41a9-bd18-62205afa5cfc.png)

Reply:
What do you think about "cuadro de actualización"(update box) or "bloque de actualización" (update block) ?
As it is not usual for final users to speak about "UI" in spanish

Reply:
What do you think about "cuadro de actualización"(update box) or "bloque de actualización" (update block) ?
As it is not usual for final users to speak about "UI" in spanish

Reply:
`bloque de actualización` sounds fine to me. Is `cuadro` really used as a term for a UI box? If yes then I am fine with both.